### ISSUE TITLE
REPL doesn't print like Julia REPL

### ISSUE THREAD

**Issue by ranjanan** [JuliaComputing/JuliaDT#13]
It doesn't print like the Julia REPL (with type, dim and pretty printing)
```
julia>a = rand(10)
[0.09295829191015548,0.26859781445460706,0.21744895390116548,0.30039385338740066,0.9149908340750528,0.735322137039141,0.020535524788270143,0.9065311794917377,0.9093804762036486,0.12773069477574928]
```

**Comment by snefru** [comment]
fixed in c828c48b1ea1a78556f5fff61ecce9482754962b
![screen shot 2016-03-28 at 09 15 55](https://cloud.githubusercontent.com/assets/1975882/14074032/cf65c8ce-f4c5-11e5-8f64-2149598fe484.png)

**Comment by ranjanan** [comment]
:+1:
